libkbfs/backpressure_disk_limiter.go

```diff
@@ -785,6 +810,18 @@ func defaultGetFreeBytesAndFiles(path string) (int64, int64, error) {
 	return int64(freeBytes), int64(freeFiles), nil
 }
 
+func (bdl *backpressureDiskLimiter) simpleByteTrackerFromType(typ diskLimitTrackerType) (
```

Comment:
The tracker talks about resources (since it can track anything), and the limiter talks about bytes. I can change it, but there is a clear separation there in my mind.